play-java-grpc-example/scripts/test-sbt

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
-
-echo "+----------------------------+"
-echo "| Executing tests using sbt  |" 
-echo "+----------------------------+"
-sbt ++$MATRIX_SCALA test docs/paradox
```

Comment:
You removed `docs/paradox`? Why?

Reply:
Yep, removed. This docs  had been publishing in Lightbend Developer Portal, but we don't publish it anymore. That why I think it isn't actual for us now and we don't need to check it during build. We should migrate this docs to Antora in the future 🤔